corehq/apps/reports/standard/forms/reports.py

```diff
@@ -58,6 +60,16 @@ def submitfilter(self):
             self._submitfilter = SubmissionTypeFilter.get_filter_toggle(self.request)
         return self._submitfilter
 
+    @property
+    @memoized
+    def selected_form_data(self):
```

Comment:
Hey @zandre-eng 

I am unable to understand the decision making in this function. can you please add a comment about it?
Why are we returning only one form even when there are multiple? And the decision making around fuzzyness?

Reply:
I've extracted the function out to a separate mixin that can be used in both reports ([fe1c14a](https://github.com/dimagi/commcare-hq/pull/34288/commits/fe1c14a8ee10412097e98246570991323c23e040)).

Reply:
Thank you for the details.
I am still unsure about the decision making within this function though I see we have not added that here so its out of scope of this PR.

Reply:
Should not this function be removed now since we included the mixin now?

Reply:
Ah, it seems this change didn't get committed. Thanks for flagging, I have removed it ([8b49b7f](https://github.com/dimagi/commcare-hq/pull/34288/commits/8b49b7fc291f5e1ff64d244f180c16c5396863ca)).